src/Messages/TSMessagesManager.m

```diff
@@ -1001,8 +1001,12 @@ - (void)processException:(NSException *)exception envelope:(OWSSignalServiceProt
       } else if ([exception.name isEqualToString:InvalidVersionException]) {
           errorMessage = [TSErrorMessage invalidVersionWithEnvelope:envelope withTransaction:transaction];
       } else if ([exception.name isEqualToString:UntrustedIdentityKeyException]) {
-          errorMessage =
-              [TSInvalidIdentityKeyReceivingErrorMessage untrustedKeyWithEnvelope:envelope withTransaction:transaction];
```

Comment:
You've deprecated `TSInvalidIdentityKeySendingErrorMessage` above by removing its initializer. 

* Can we deprecate `TSInvalidIdentityKeyReceivingErrorMessage` too?
* Can we mark the `TSErrorMessageWrongTrustedIdentityKey` enum value as "unused"?